docs/static/releasenotes.asciidoc

```diff
@@ -0,0 +1,66 @@
+[[releasenotes]]
+== Release Notes
+
+This section summarizes the changes in the following releases:
+
+* <<logstash-5-3-0,Logstash 5.3.0>>
+
+[[logstash-5-3-0]]
+=== Logstash 5.3.0 Release Notes
+
+
+* Persistent queues:
+    ** Changed the default queue location on disk to include the pipeline's ID in the path hierarchy.
+       By default, the queue is now created under <path.data>/queue/main. This breaking change was made to
+       accommodate an upcoming feature where multiple, isolated pipelines could be run on the same Logstash
+       instance.
+    ** Added a _recovery_ process that runs when Logstash starts up to recover data written to the persistent
+       queue, but not yet checkpoint'd. This is useful in a situation when the input had written data to the queue,
+       but Logstash crashed before writing to the checkpoint file.
+    ** Added exclusive access to the persistent queue on disk, as defined by the path.queue setting. Using a file
```

Comment:
Suggest changing this to say, "Using a file lock guards against corruption by ensuring that only a single Logstash instance has access to write to the queue on the same path."